app/models/target.rb

```diff
@@ -22,8 +22,10 @@ class Target < ApplicationRecord
   acts_as_mappable lat_column_name: :latitude,
                    lng_column_name: :longitude
 
-  belongs_to :topic
   belongs_to :user
+  belongs_to :topic
+  has_many :matches, dependent: :destroy
```

Comment:
There should be tests for these relations in the spec/models/target_spec.rb file